examples/server-side-rendering/src/app/app.component.ts

```diff
@@ -69,16 +89,18 @@ export class AppComponent {
     private injector: Injector,
     @Inject(PLATFORM_ID) private platformId: Object
   ) {
+    const routing = { router: history(), stateMapping: simple() };
+
     const req = isPlatformServer(this.platformId)
       ? this.injector.get('request')
       : undefined;
 
-    const searchParameters = parseServerRequest(req);
+    const initialUiState = parseServerRequest(req, routing);
```

Comment:
note that we are passing the routing argument, not the processed routing, since the InstantSearch instance doesn't yet exist.

Reply:
This means that I'm assuming the `history` router here, ideally the solution would work with any router?

Reply:
We don't need this anymore without using routing.